docs/tutorials/tutorials/OpenVINO_EP_samples/cleanroom_worker_safety_python.md

```diff
@@ -0,0 +1,45 @@
+This is a python sample application for demoing industrial/clean room use case using ONNXRuntime OpenVINO Execution Provider.
+
+1. The Object detection sample uses a pre-trained Tiny YOLOv2 Deep Learning ONNX Model from Intel. 
```

Comment:
It seems the folder is missing the onnx model https://github.com/microsoft/onnxruntime/tree/eb7c99e796be2a1ee49cf2d38faa19e4f9dd2a64/samples/python/OpenVINO_EP_samples/cleanroom_worker_safety.

Reply:
yeah even though I am pushing the onnx model for the clean room sample it is not getting reflected in microsoft/onnxruntime repository. so just created a tar file of the model and pushed it.

Added subsequent steps of how user can untar the file to get the onnx model in the documentation.